index.html

```diff
@@ -5,11 +5,92 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="css/style.css">
     <title>Responsive Cake webpage</title>
     <!-- Add a link to your css file here -->
 </head>
 
 <body>
+    <header>
+        <div class="image-frame">
+            <img class="img-header" src="img/photo-cake3.jpg" alt="Header Photo">
+        </div>
+        <div class="right-header">
+            <div  class="menu-frame">
+                <img  class="menu-header" src="img/menu-hamburger.svg" alt="Menu">
+                <div class="div-free">
+                    <p></p>
+                </div>
+            </div>
+            <article class="header-text">
+                Treat yourself to the sweetest and most delicious taste.
+            </article>
+        </div>
+    </header>
+    <main>
+        <nav class="the-nav">
+            <ul class="un-order-list">
+                <div class="free-li-dev"></div>
+                <div class="rectangle">
+                    <li>HOME</li>
```

Comment:
Couldn't you add a class and style the `li` directly, removing the need for the div?
Or do something like:

`<li><p>HOME</p></li>`